terraform/deployments/tfc-configuration/variables-integration.tf

```diff
@@ -102,6 +102,22 @@ module "variable-set-rds-integration" {
         freestoragespace_threshold   = 10737418240
       }
 
+      chat = {
+        engine         = "postgres"
+        engine_version = "16"
+        engine_params = {
+          log_min_duration_statement = { value = 10000 }
+          log_statement              = { value = "all" }
+          deadlock_timeout           = { value = 2500 }
+          log_lock_waits             = { value = 1 }
+        }
+        engine_params_family         = "postgres16"
+        name                         = "chat"
+        allocated_storage            = 100
```

Comment:
You could probably reduce this a bit. Can always set autoscaling for the storage.